grommet/grommet
# Add a11ytitle support for Grid

#### What does this PR do?
This PR allows users to provide an a11yTitle to Grid. Even though the prop was listed in the documentation, when users provided this prop it was not appearing in the DOM.

#### Where should the reviewer start?
src/js/components/Grid/Grid.js

#### What testing has been done on this PR?
This change has been tested in a local project by adding the prop `a11yTitle="My Grid"` to a Grid. Using the inspector tools, I could see that the aria-label was added. I also turned on voiceover to check that a screen reader would be able to pick up on the label. Additionally, I've added a new jest test that checks that the aria-label is created.

#### How should this be manually tested?
In a local project create a grid and add the `a11yTitle` prop. Using the inspector tools, check that the aria-label appears in the DOM.

#### Any background context you want to provide?
While the docs said you could supply an a11yTitle to Grid, the code wasn't supporting this. So, when users tried to add this prop, it wouldn't have an effect on their code.

#### What are the relevant issues?
#3517 

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No.

#### Should this PR be mentioned in the release notes?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes, this is backwards compatible.
